guides/source/active_support_instrumentation.md

```diff
@@ -543,6 +543,13 @@ Active Storage
 | `:key`       | Secure token        |
 | `:service`   | Name of the service |
 
+### service_delete_prefixed.active_storage
+
+| Key          | Value                            |
+| ------------ | -------------------------------- |
+| `:prefix`    | The key starting with the prefix |
```

Comment:
The Value here seems off. What do you think about changing it to “Key prefix”?

Reply:
@georgeclaghorn Thanks for the comment. I feel it so simply. I'll modify it soon.